backend/routes/api.php

```diff
@@ -19,6 +19,8 @@
 Route::post('/register', [AuthController::class, 'register']);
 Route::post('/login', [AuthController::class, 'login']);
 Route::get('/categories', [CategoryController::class, 'index']);
+Route::get('/categories/words/{slug}', [CategoryController::class, 'getWordsBySlug']);
+Route::get('/categories/words/{slug}/{id}', [CategoryController::class, 'getWordsBySlugAndId']);
```

Comment:
Does this have similiar output?

if you want to use the `slug` as params,  read about [customizing the key name](https://laravel.com/docs/9.x/routing#customizing-the-key)